lib/streamlit/runtime/caching/cache_utils.py

```diff
@@ -246,7 +252,7 @@ def get_or_create_cached_value():
 
             return return_value
 
-        if cached_func.show_spinner:
+        if cached_func.show_spinner or isinstance(cached_func.show_spinner, str):
```

Comment:
We want to explicitly support the usage of empty string or?

Reply:
Good question!

I deliberately did that to support empty string, but this is not something explicitly mentioned in Product Spec
After an `empty label` project I think it is often a good idea to support minimalistic elements (like just spinner animation without label. 

But let me check this with Johannes, to have his input for this too!